components/BhilaeeHub.module.css

```
@@ -0,0 +1,533 @@
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    padding: 10px 18px;
+    border: 1px solid var(--border-color);
+    border-radius: 50px;
+    background: var(--card-bg);
+    color: var(--text-color);
+    cursor: pointer;
+    font-size: 0.9rem;
+    font-weight: 600;
+    font-family: 'Inter', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)